src/Backend/Core/Layout/Templates/header.html.twig

```diff
@@ -48,3 +48,9 @@
     </li>
   </ul>
 </div>
+
+{% include "/Layout/Templates/messaging.html.twig" %}
+
+<div id="ajaxSpinner" style="position: fixed; top: 10px; right: 10px; display: none;">
```

Comment:
This seems rather dirty, perhaps we could have a frontender have a poke at this?